base/common/CMakeLists.txt

```diff
@@ -66,7 +66,10 @@ if(WITH_TEST)
             pki-certsrv-jar
     )
 
-    # TODO: create CMake function to find all JUnit test classes
+    execute_process(COMMAND bash "-c" "grep -ilR @Test ${PROJECT_SOURCE_DIR} | cut -d':' -f1 | awk -F '/src/test/java/' '{ print $2 }' | sed 's/.java/;/g' | sed 's!/!.!g' | tr -d '\n'" OUTPUT_VARIABLE DISCOVERED_TESTS)
+    execute_process(COMMAND bash "-c" "echo ${DISCOVERED_TESTS}")
+    execute_process(COMMAND bash "-c" "pwd")
```

Comment:
@edewata ignore these two lines, they were for debugging purposes - I shall remove them before merging.